roles/manage_touchstone_tools/tasks/stop_db_stats.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Stop ts pgsql-stat
+  ansible.builtin.shell: |
+    set -o pipefail
+    ts pgsql-stat \
+      -o {{ ts_output_dir }} \
```

Comment:
Same comment here, about only needing the specify the data collection directory.